tests/apollo/util/operator.py

```diff
@@ -279,3 +289,11 @@ async def add_remove_with_wedge_status(self, quorum=None):
             quorum = bft_client.MofNQuorum.All(self.client.config, [r for r in range(self.config.n)])
         reconf_msg = self._construct_reconfiguration_addRemoveWithWedgeStatus_command()
         return await self.client.read(reconf_msg.serialize(), m_of_n_quorum=quorum, reconfiguration=True)
+
+    async def get_dbcheckpoint_info_request(self):
+        cpinfo_msg = self._construct_reconfiguration_get_dbcheckpoint_info_request()
+        return await self.client.read(cpinfo_msg.serialize(), reconfiguration=True)
+    
+    async def create_dbcheckpoint_cmd(self):
+        cp = self._construct_reconfiguration_create_dbcheckpoint_command()
+        return await self.client.read(cp.serialize(), reconfiguration=True)
```

Comment:
why create db_checkpoint is a read request?

Reply:
fixed